ui/src/app/app-routing.tsx

```diff
@@ -243,170 +216,9 @@ export const AppRoutingComponent: React.FunctionComponent<RoutingProps> = ({onSi
               />}
           />
           <AppRoute
-              path='/workspaces/:ns/:wsid/about'
```

Comment:
Breaking this up might be preferable, but if we'd prefer to keep it together another option is to pass in the inner routes as React children to the Workspace wrapper.